recipe/meta.yaml

```diff
@@ -13,15 +13,17 @@ source:
   md5: 8987b9a3e3cd79218a0a423b21c8e4de
 
 build:
-  number: 200
+  number: 201
   # We lack openblas on Windows, and therefore can't build scipy there either currently.
   skip: true  # [win]
   features:
     - blas_{{ variant }}
 
 requirements:
   build:
+    - toolchain
     - gcc
+    - cloog 0.18.0 10
```

Comment:
Yeah, it's not. It's related to the fact that `gcc` package does not pin `isl` correctly. As we have a newer version of `isl` than `defaults`, this causes problems with the compiler. We have tried to rebuild the compiler, but do not have sufficient build time ATM.

We came up with a fragile workaround that used to work by default for awhile using this `cloog` package. However, it seems changes in the solver have caused that to fail. Basically this `cloog` package (which we build [here]( https://github.com/conda-forge/cloog-feedstock )) takes care of pinning `isl` correctly as `gcc` does not. To get a better idea of what is going on please take a look at this issue ( https://github.com/conda-forge/staged-recipes/issues/436 ) and this issue ( https://github.com/conda-forge/isl-feedstock/issues/3 ). It looks like Continuum will rebuild the `gcc` package, but we don't know the time frame of that change yet. So, this is simply the explicit form the previously implicit workaround. As it is explicit, it is not fragile in the same way.